DependencyInjection/HttplugExtension.php

```diff
@@ -54,6 +66,10 @@ protected function configureClients(ContainerBuilder $container, array $config)
                 $first = $name;
             }
 
+            if (isset($config['_inject_collector_plugin'])) {
+                array_unshift($arguments['plugins'], 'httplug.collector.history_plugin');
```

Comment:
If you have the CachePlugin and put the HistoryPlugin last you will not see anything... But that might be okey, since no request was actually made.